Metaclasses/Metaclasses.tex

```diff
@@ -231,7 +231,7 @@ \section{Every class is an instance of a metaclass}
 TranslucentColor allSuperclasses  --> an OrderedCollection(Color Object ProtoObject)
 TranslucentColor instVarNames     --> #('alpha')
 TranslucentColor allInstVarNames --> #('rgb' 'cachedDepth' 'cachedBitPattern' 'alpha')
-TranslucentColor selectors sorted --> #(#addName: #alpha #alpha: #asHTMLColor #asNontranslucentColor #balancedPatternForDepth: #bitPatternForDepth: #convertToCurrentVersion:refStream: #hash #isOpaque #isTranslucent #isTranslucentColor #isTransparent #name #pixelValueForDepth: #pixelWordForDepth: #privateAlpha #scaledPixelValue32 #setRgb:alpha: #storeArrayValuesOn: #storeOn:)
+TranslucentColor selectors sorted --> #(#addName: #alpha #alpha: #asHTMLColor #asNontranslucentColor #balancedPatternForDepth: #bitPatternForDepth: #convertToCurrentVersion:refStream: #hash #isOpaque #isTranslucent #isTranslucentColor #isTransparent #name #pixelValueForDepth: #pixelWordForDepth: !\SqVersionSwitch{6.0}{\#printOn: }{}!#privateAlpha #scaledPixelValue32 #setRgb:alpha: #storeArrayValuesOn: #storeOn:)
```

Comment:
Hm ... this does not look very beautiful:

![image](https://user-images.githubusercontent.com/38782922/107962626-74849680-6fa7-11eb-8907-e0a96e07c346.png)

Reply:
This indeed seems to be a problem with a strong connection to the `\lstlisting` implementation. I created a relevant question for it on tex.stackexchange: [lstlisting: Unexpected newline/missing background color after trailing space in escape section](https://tex.stackexchange.com/q/583523/221054)

Reply:
Let's wait a few days for an answer on tex.se and otherwise get this merged with the current state. This very much looks like a bug to me so we shouldn't let it block two other PRs IMO.

Reply:
Ha! Fixed due to a hack proposed by Ulrike Fischer on tex.se via ef59c1a.

Current build for 5.3:
![image](https://user-images.githubusercontent.com/38782922/108121808-a9761380-70a3-11eb-98a9-e6eca20b929f.png)

Current build for Trunk:
![image](https://user-images.githubusercontent.com/38782922/108121827-ada23100-70a3-11eb-8078-11749f6f9d99.png)